app/jobs/clean_up_expired_pushes_after_duration_job.rb

```diff
@@ -0,0 +1,62 @@
+class CleanUpExpiredPushesAfterDurationJob < ApplicationJob
```

Comment:
Let's name this job `PurgeExpiredPushesJob` to more closely match the yaml setting.

Plus using `purge` in the name also aligns a bit better with the already existing jobs:

purge_unattached_blobs_job.rb
clean_up_pushes_job.rb
expire_pushes_job.rb